### ISSUE TITLE
Game crashes randomly

### ISSUE THREAD

**Issue by Hamstergg10** [Gaztoof/GMod-SDK#38]
Sometimes my game crash randomly while i'm playing. How i fix it?

**Comment by 1230912zdfol** [comment]
are you sure your not repeatly defining A material?

**Comment by ljboxf** [comment]
Yeah I've been getting this issue since the latest update of this..

**Comment by 1230912zdfol** [comment]
I recommend using the visual studio debugger and seeing why its crashing!

**Comment by oops136** [comment]
still a relevant issue; what version did this crash start occurring, if you know?
